physionet-django/project/serializers.py

```diff
@@ -0,0 +1,34 @@
+# from user.models import User
```

Comment:
Please could you remove these commented lines?
`# from user.models import User`
`# adding License & Dua for details`